### ISSUE TITLE
Dark font colors at information bar

### ISSUE THREAD

**Issue by parkmino** [CDrummond/cantata#1381]
Hard to read it and clean build does not solve this problem.

![2018-10-01-132754_1600x900_scrot](https://user-images.githubusercontent.com/12131158/46269732-efd38200-c57d-11e8-819b-05891e38aaaf.png)

**Version Info**
 - OS: Ubuntu 18.04
 - Cantata Version: git
 - MPD Version: 0.21git
 - CMake options (other than default): -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DENABLE_AVAHI=OFF -DENABLE_LIBVLC=ON

**Comment by CDrummond** [comment]
That's a theme issue. Cantata sets the colour of these labels from the toolbar button text colour. You have the light text colour set for the rest of your style - I assume this is overriding KVantum, which should set the text colour for this 'toolbar' to white.

**Comment by parkmino** [comment]
It is weird that time colour turns to bright after mouseover.

**Comment by CDrummond** [comment]
Just a guess, but are you using `qt5ct`? This delays setting the palette, and breaks Cantata. As a quick hack (and this is **not** a fix); edit widgets/nowplayingwidget.cpp and at line 505 insert 'initColors()' - ie. change from:

```
void NowPlayingWidget::resizeEvent(QResizeEvent *ev)
{
    QWidget::resizeEvent(ev);
    controlWidgets();
}
```

to:

```
void NowPlayingWidget::resizeEvent(QResizeEvent *ev)
{
    QWidget::resizeEvent(ev);
    initColors();
    controlWidgets();
}
```

Rebuild, and restart Cantata. To trigger changes, resize the window.

This is not a real fix, as this would be called on each resize. But if it works, then I'll set the colors on a showEvent instead.

**Comment by parkmino** [comment]
Yes, I am using qt5ct and try this once the build error is resolved.

**Comment by CDrummond** [comment]
As to the build error - just edit mpd-interface/httpstream.cpp and change:

```
void HttpStream::bufferingProgress(int progress)
{
    MPDStatus * const status = MPDStatus::self();
    if (status->state() == MPDState_Playing) {
        if (progress == 100) {
            player->play();
        } else {
            player->pause();
        }
    }
}
```

to 
```
void HttpStream::bufferingProgress(int progress)
{
   Q_UNUSED(progress)
}
```

**Comment by parkmino** [comment]
Yeah, both patches work!

**Comment by CDrummond** [comment]
Please try with latest git/master.

**Comment by parkmino** [comment]
Still greyed out

**Comment by CDrummond** [comment]
Hmm... That means no palette change. If you edit `gui/mainwindow.cpp` and change:

```

void MainWindow::showEvent(QShowEvent *event)
{
    #if defined Q_OS_WIN
    if (!thumbnailTooolbar) {
        thumbnailTooolbar=new ThumbnailToolBar(this);
    }
    #endif
    QMainWindow::showEvent(event);
    controlView();
}
```

to:
```

void MainWindow::showEvent(QShowEvent *event)
{
    #if defined Q_OS_WIN
    if (!thumbnailTooolbar) {
        thumbnailTooolbar=new ThumbnailToolBar(this);
    }
    #endif
    QMainWindow::showEvent(event);
    controlView();
    paletteChanged();
}
```

Does this help?

**Comment by parkmino** [comment]
It does not work either.

**Comment by CDrummond** [comment]
Hmmm... That makes no sense. `MainWindow::showEvent` is called (or should be) when the window is shown. This then calls `paletteChanged()` which calls the `initColors()` to set the colours. This _should_ have the same effect as before...

**Comment by CDrummond** [comment]
Should be fixed now. Please update and confirm.
